### PR TITLE
Bug 1698548 - Link to Talos test documentation

### PR DESCRIPTION
**Pull request by beatrice-acasandrei**

### CODE REVIEW

File: ui/perfherder/graphs/LegendCard.jsx [hunk 1]
```diff
@@ -162,6 +164,9 @@ const LegendCard = ({
 
           {series.name}
         </Button>
+        <div className="small">
+          {framework === 'talos' && <a href={getTalosDocsURL(suite)}>(docs)</a>}
```
Comment (ionutgoldan) [inline ui/perfherder/graphs/LegendCard.jsx:168]: It'd be more consistent to provide the link right after the series name (immediately above).

File: ui/perfherder/compare/CompareTable.jsx [hunk 2]
```diff
@@ -172,7 +176,10 @@ export default class CompareTable extends React.Component {
         <thead>
           <tr className="subtest-header bg-lightgray">
             <th className="text-left">
-              <span>{testName}</span>
+              <span>
+                {testName}
+                {framework === 'talos' && <a href={url}> (docs)</a>}
```
Comment (ionutgoldan) [inline ui/perfherder/compare/CompareTable.jsx:181]: Could you look for a way to display the `(docs)` link with blue, like normal anchor links look like?

**Review by ionutgoldan — CHANGES_REQUESTED** [review]
I also noticed the following error when clicking on a subtests hyperlink from Compare view:

```javascript
Something went wrong TypeError: dropdownOptions[0] is undefined
```

File: ui/perfherder/compare/CompareTable.jsx [hunk 3]
```diff
@@ -172,7 +176,15 @@ export default class CompareTable extends React.Component {
         <thead>
           <tr className="subtest-header bg-lightgray">
             <th className="text-left">
-              <span>{testName}</span>
+              <span>
+                {testName}
+                {frameworkName === 'talos' && (
+                  <a href={url} style={{ color: '#23527c' }}>
+                    {' '}
```
Comment (airimovici) [inline ui/perfherder/compare/CompareTable.jsx:183]: This space can be moved outside outside the `<a>`. Just before it, so we avoid having the space underlined on hover

File: ui/perfherder/compare/CompareTable.jsx [hunk 4]
```diff
@@ -172,7 +176,15 @@ export default class CompareTable extends React.Component {
         <thead>
           <tr className="subtest-header bg-lightgray">
             <th className="text-left">
-              <span>{testName}</span>
+              <span>
+                {testName}
+                {frameworkName === 'talos' && (
+                  <a href={url} style={{ color: '#23527c' }}>
```
Comment (airimovici) [inline ui/perfherder/compare/CompareTable.jsx:182]: we should have the style defined in `perf.css` file.

https://github.com/mozilla/treeherder/blob/master/ui/css/perf.css#L312

```
.compare-table .result-links a,
.compare-table .subtest-header a {
  color: #23527c;
}
```

**Review by airimovici — CHANGES_REQUESTED** [review]

**Review by airimovici — APPROVED** [review]

**Comment by ionutgoldan** [comment]
I found a weird bug.

### Steps to reproduce

1. Go to Alerts view.
2. Select a random alert summary.
3. Add some random notes to it.
4. Click on the summary' s header link.

#### Expected results
User gets redirected to that single summary.

#### Actual results
Following Javascript error is dumped in the UI:

```javascript
Something went wrong TypeError: alert.title is undefined
```

**Review by ionutgoldan — CHANGES_REQUESTED** [review]
We need to fix the reported bug & do more thorough manual tests.

**Pull request closed without merging**